net/shorewall/files/hostname

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+cat /proc/sys/kernel/hostname
```

Comment:
Is there a reason to not use the `hostname` command here?

Reply:
Because hostname is not installed by default.

Reply:
What about ``uci get system.@system[0].hostname``